torch_xla/csrc/runtime/pjrt_computation_client.cc

```diff
@@ -458,6 +458,16 @@ std::vector<ComputationClient::DataPtr> PjRtComputationClient::ReshardData(
   return resharded_results;
 }
 
+std::uintptr_t PjRtComputationClient::UnsafeBufferPointer(
+    const DataPtr handle) {
+  std::shared_ptr<PjRtData> pjrt_data = ReplicateShardedData(handle);
```

Comment:
why do you need to call `ReplicateShardedData`? Is it to handle the SPMD cases?

Reply:
I don't intend to handle the SPMD case atm. Let me change it.